### ISSUE TITLE
Real/Fake Accuracy

### ISSUE THREAD

**Issue by Queuecumber** [AlexiaJM/RelativisticGAN#21]
If I want to compute the real/fake accuracy of the discriminator for a batch using this method, do I need to do:

```
real_accuracy = sigmoid(real_logits - mean(fake_logits)) >= 0.5
fake_accuracy = sigmoid(fake_logits - mean(real_logits)) < 0.5
```

e.g. the same input that I would give to the BCE with logits loss goes to the sigmoid

Or do I do the standard GAN thing and compute

```
real_accuracy = sigmoid(real_logits) >= 0.5
fake_accuracy = sigmoid(fake_logits) < 0.5
```

**Comment by AlexiaJM** [comment]
There's no concept of accuracy in the traditional sense here, but if you want something as close possible with RaSGAN, your first approach is how I would do it. The second approach won't make sense.

**Comment by Queuecumber** [comment]
Thanks, usually when I train a GAN I look for the accuracy to sit around ~80% for both real and fake to know if it's doing ok, is there a surrogate metric you'd recommend for the RA loss?
